react/week4/src/components/Repo.js

```diff
@@ -0,0 +1,47 @@
+
+import React,{useEffect,useState} from 'react';
+import  RepoList from './RepoList';
+
+const Repo = () => {
+  const [users, setUsers] = useState([]);
+  const [error, setError] = useState(null);
+  
+  useEffect(() => { 
+    ;async function fetchData() {
+    try {
+        const response = await fetch(`https://api.github.com/users/mahieakhtar/repos`)
+        const data = await response.json();
+        console.log(data);
+        setUsers(data);
+        
+    } catch (error) {
+        setError(error);
+        }
+    }
+    fetchData()
+}, []);
+/*useEffect(()=>{ 
+  async function fetchData() {
+    const response = await fetch(`https://api.github.com/users/mahieakhtar/repos`)
+    const data = await response.json()
+    console.log(data)
+    setUsers(data)
+    
+  }
+  fetchData()
+},[]);*/
+
+    return (
+        <div>
+           {users!=null && users.map(user=>(
+             <RepoList 
+              key={user.id}
+              login={user.owner.login}
+              fullname = {user.full_name}/>  
```

Comment:
You don't need spaces before and after `=` I was you do that in different places, please update it everywhere
```suggestion
              fullname={user.full_name}/>  
```